static/docs/commands-reference/fetch.md

```diff
@@ -1,62 +1,287 @@
+
+This project comes with a predefined S3 [remote
+storage](https://man.dvc.org/remote). We can now just run `dvc fetch`
+that will download the most recent `model.pkl`, `data.xml`, and other files that
+are under DVC control into our local cache:
+
+```shell
+    $ ldvc status -c    <- compares local cache vs default remote
 
-```dvc
     $ dvc fetch
+    ...
+    (2/6): [##############################] 100% data/features/test.pkl
+    (3/6): [##############################] 100% model.pkl
+    (4/6): [##############################] 100% data/features/train.pkl
+    ...
+
+    $ tree .dvc
+    .dvc
+    ├── cache           <- dir .dvc/cache was created and populated
+    │   ├── 38
+    │   │   └── 63d0e317dee0a55c4e59d2ec0eef33
+    │   ├── 42
+    │   │   └── c7025fc0edeb174069280d17add2d4.dir
+    │   ├── ...
+    ├── config
+    ├── ...
+```
+
+As seen above, used without arguments, `dvc fetch` downloads all assets needed
+by all DVC files in∑ the current branch, including for directories. The checksums
+`3863d0e317dee0a55c4e59d2ec0eef33` and `42c7025fc0edeb174069280d17add2d4`
+correspond to the `model.pkl` file and `data/features/` directory, respectively.
+
+```shell
+    $ dvc checkout      <- links files from local cache to workspace
+    Checking out '{'scheme': 'local', 'path': '.../example-get-started/model.pkl'}' with cache '3863d0e317dee0a55c4e59d2ec0eef33'.
+    Checking out '{'scheme': 'local', 'path': '.../example-get-started/data/features'}' with cache '42c7025fc0edeb174069280d17add2d4.dir'.
+    ...
+```
+
+## Examples: Specific stages
+
+Fetch only outputs of a specific stage bt specifying its DVC file (target
+stage):
 
-    (1/8): [#################################] 100% images/0001.jpg
-    (2/8): [#################################] 100% images/0002.jpg
-    (3/8): [#################################] 100% images/0003.jpg
-    (4/8): [#################################] 100% images/0004.jpg
-    (5/8): [#################################] 100% images/0005.jpg
-    (6/8): [#################################] 100% images/0006.jpg
-    (7/8): [#################################] 100% images/0007.jpg
-    (8/8): [#################################] 100% model.pkl
+```shell
+    $ dvc fetch prepare.dvc
+    ...
+    (1/2): [##############################] 100% data/prepared/test.tsv
+    (2/2): [##############################] 100% data/prepared/train.tsv
+
+    $ tree .dvc/cache      
+.dvc/cache
+├── 42
+│   └── c7025fc0edeb174069280d17add2d4.dir
+├── 58
+│   └── 245acfdc65b519c44e37f7cce12931
+├── 68
+│   └── 36f797f3924fb46fcfd6b9f6aa6416.dir
+└── 9d
+    └── 603888ec04a6e75a560df8678317fb
 ```
 
-Fetch outputs of a specific dvc file:
+Cache entries for the necessary directories, as well as the actual
+`data/prepared/test.tsv` and `data/prepared/train.tsv` files were download,
+checksums shown above.
+
+## Examples: With dependencies
+
+Demonstrating the `--with-deps` flag requires a larger example. First, assume
```

Comment:
I think it should be easy to use the same example now from above.

Reply:
Oh yeah I guessed I copied this from `dvc pull` as requested earlier but it doesn't make sense any more. Will update

Reply:
Updated, please check